beacon-chain/core/helpers/validators_test.go

```diff
@@ -405,3 +405,126 @@ func TestActiveValidatorIndices(t *testing.T) {
 		})
 	}
 }
+
+func TestComputeProposerIndex(t *testing.T) {
+	seed := bytesutil.ToBytes32([]byte("seed"))
+	type args struct {
+		validators []*ethpb.Validator
+		indices    []uint64
+		seed       [32]byte
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    uint64
+		wantErr bool
+	}{
+		{
```

Comment:
can you also one more test case that closely mimics for what we observed in our testnet ?  example: `validator > 600` isnt selected for proposal

Reply:
There is one for that, I forgot to rename it, but see `second_half_active` which is the regression that only the first 5 would have been selected when the latter 5 were the only ones active.